apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx

```diff
@@ -86,10 +88,15 @@ const EditMusicLevelData: React.FunctionComponent<EditMusicLevelDataProps> = ({
             ? a.artist.localeCompare(b.artist) || a.name.localeCompare(b.name)
             : 0
         )
-        ?.map(({name, id, artist}) => ({
-          value: id,
-          text: `${artist} - ${name}`,
-        })),
+        ?.map(({name, id, artist, bpm, sounds}) => {
+          // Use the pack bpm if present, or the bpm of the first sound that has one.
+          const packTempo =
+            bpm || sounds?.find(sound => sound.bpm)?.bpm || DEFAULT_BPM;
+          return {
+            value: id,
+            text: `[${packTempo}] ${artist} - ${name}`,
```

Comment:
Should we go ahead and display the key here too while we're at it, in case it becomes useful in the future? Something like `"Aloe Blacc - I Need a Dollar [95 / B]"`

We can use the `getNoteName` util from `Notes.ts` to get the display note name

Reply:
Katie said:

> That could be helpful! But want to be sensitive to overloading the menu. I imagine using that information more when we’re designing a lesson that has more of a focus on music education.

Given that sensitivity, I'm going to leave this branch as is. It would be a very simple follow-up if it becomes desired though!

Reply:
Cool makes sense - just noting that key information may also be helpful in determining how much the Code.org sounds will be pitch-shifted (i.e. a key center very far from C like F/F#/G, will have a lot of stretching, vs closer keys like B/C#/D will have less stretching)